src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -553,8 +550,7 @@ public void removeDatabaseChangeListener(DatabaseChangeListener l) {
      */
     public static String getResolvedField(String field, BibEntry entry, BibDatabase database) {
         if ("bibtextype".equals(field)) {
-            TypedBibEntry typedEntry = new TypedBibEntry(entry, Optional.ofNullable(database));
-            return typedEntry.getTypeForDisplay();
+            return EntryUtil.capitalizeFirst(entry.getType());
```

Comment:
Not sure what is correct here. getTypeForDisplay is slightly different from captializeFirst (for example it returns InProccedings instead of Inproceedings). Maybe even returning the type as a plain lowercase string is enough/better?